test/core.jl

```diff
@@ -3949,6 +3949,40 @@ function f16158(x)
 end
 @test f16158("abc") == "abcaba"
 
+# LLVM verifier error for noreturn function
+# These are only meaningful on debug build with verifier on.
+let
+    local f1, f2, f3, g
+    function f1(a)
+        if a
+            b = (a + 1, 2)
+        else
+            throw(DivideError())
+        end
+        b[1]
+    end
+    code_llvm(DevNull, f1, Tuple{Bool})
+    # @noinline g() = error()
```

Comment:
The test should still pass for release build. Just that this won't be testing anything particularly useful.